src/scene/batching/batch-manager.js

```diff
@@ -142,8 +142,8 @@ class BatchManager {
         }
 
         if (this._batchGroups[id]) {
-            Debug.error(`batch group with id ${id} already exists`);
-            return;
+            Debug.error(`Batch group with id ${id} already exists.`);
+            return undefined;
```

Comment:
maybe it could return `this._batchGroups[id]` so that the code would still work

Reply:
Again, it'd be changing functionality. I'd rather address that separately.